external/ag-website-shared/src/components/license-pricing/license-pricing.module.scss

```diff
@@ -15,17 +15,26 @@
 }
 
 .introSection {
-    width: 80%;
-    margin: 0 auto;
     display: flex;
     align-items: center;
     flex-direction: column;
-    margin-bottom: $spacing-size-8;
+    padding-top: $spacing-size-12 + ($spacing-size-1 / 2);
+    background: linear-gradient(
+        to bottom,
+        transparent,
+        color-mix(in srgb, var(--background) 5%, transparent) ($spacing-size-12 + 64px / 2),
+        transparent 0
+    );
+    --background: var(--color-bg-brand-solid);
```

Comment:
Minor nitpick. Define CSS vars at the top of declaration block.

Reply:
Updated